server/bin/www

```diff
@@ -51,7 +51,7 @@ const authSocket = async (socket, next) => {
   return next();
 };
 
-io.use(authSocket);
+// io.use(authSocket);
```

Comment:
Remove if unused!

Reply:
This line was created by a teammate, and it's throwing errors so I commented it, but I don't want to delete it because is his code. What should I do in this case?

Reply:
We need to get out of the habit of just commenting code that doesn't work. Discuss with your team mate why it's not working and work together to get a fix for it.

Reply:
I asked my team mate what about this issue and he will be create a new PR soon.